scripts/ci/vercel-deploy.sh

```
@@ -0,0 +1,7 @@
+#!/bin/bash
+pushd .
+cd ./packages/backend
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)